gpbft/gpbft.go

```diff
@@ -373,7 +381,8 @@ func (i *instance) receiveOne(msg *GMessage) (bool, error) {
 	msgRound := i.getRound(msg.Vote.Round)
 	switch msg.Vote.Step {
 	case QUALITY_PHASE:
-		// Receive each prefix of the proposal independently.
+		// Receive each prefix of the proposal independently, which is accepted at any
+		// round/phase.
 		i.quality.ReceiveEachPrefix(msg.Sender, msg.Vote.Value)
```

Comment:
I am not sure if we should, or at least there is no harm in proceeding, because if we are in CONVERGE phase then the processing of a late-arriving QUALITY message may expand candidates that [could result in adopting proposal after converge](https://github.com/filecoin-project/go-f3/blob/c0765caf1812cb7db0e1a7e4c0759f309dc53c51/gpbft/gpbft.go#L570). 

WDYT? does that make sense?

Reply:
makes sense to me

Reply:
I have gone back and forth on this one a few times. TBD in standup.

Reply:
Discussed sync. Because of the timeout in CONVERGE it makes no difference. hence keeping the exit early as suggested by Steven